BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationProcessor.swift

```diff
@@ -93,6 +95,10 @@ class CompleteRegistrationProcessor: StateProcessor<
             state.isCheckDataBreachesToggleOn = newValue
         case let .togglePasswordVisibility(newValue):
             state.arePasswordsVisible = newValue
+        case .learnMoreTapped:
+            break
+        case .preventAccountLockTapped:
```

Comment:
these will be covered in the follow up PRs that shannon has in draft.  Going to push updates to those current PRs to get these tested accordingly.